### ISSUE TITLE
[Question] Upgrade 4.4.0 to 5.0  

### ISSUE THREAD

**Issue by manwenick** [YetiForceCompany/YetiForceCRM#10893]
### Description
Hello everyone, 
I tried to upgrade from 4.4.0 to 5.0 with this file 4.4.0_to_5.0.0_v5.zip
Now I have Failure Screen:
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'default' in 'where clause' Failed to prepare SQL: SELECT * FROM `s_yf_companies` WHERE `default`=:qp0 LIMIT 1
Have read a lot of forums, but unfortunately didn't found a solution. 
What can the problem be?

**Comment by davide-alghi** [comment]
Hi manwenick,
that's a known problem.
try to read following open / closed issues

https://github.com/YetiForceCompany/YetiForceCRM/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+SQLSTATE%5B42S22%5D%3A+Column+not+found

**Comment by manwenick** [comment]
@davide-alghi hello Davide and thank you for quick response! 
I restored the db as it was descripted in the link you provided and went few times the upgrade process. 
Now I have another failure message:
SQLSTATE[HY000]: General error: 1366 Incorrect DECIMAL value: '0' for column '' at row -1 The SQL being executed was: ALTER TABLE `vtiger_project` CHANGE `progress` `progress` decimal(5,2)

**Comment by davide-alghi** [comment]
I'm sure you are working on an YF test version not production one :)
Please create a document with all change and actions. Before go into production you have to check if all data are correctly migrated: try to use the migrated version and only when you are sure that all migrated is corrected and working fine, then go into production.

anyway ...

about the last error (ALTER TABLE vtiger_project CHANGE ....), it seems that your MySql server is working in strict mode (sql_mode): please exec following (I presume that you have a single LAMP server).

- connect via ssh to your server
- login mysql client as **root** (or user with all privilegies)
- run this query ```SELECT @@sql_mode;```
- and also this ```SELECT @@GLOBAL.sql_mode;```

_... and report here results_

alternatively, if you cannot connect via ssh, run the previous queries via phpMyAdmin (as root)

**Comment by manwenick** [comment]
Hello @davide-alghi 

thank you for advise! Yes, I do all this on "DEV" server. My "PROD" server is still on 4.4.0. Only if I am sure everything works fine I will do it on PROD. :)

I tried it but I had the same error code. 
I did following: 
SSH: mysql 
SET GLOBAL sql_mode = '';

This has work for me and for the upgrade. 
I will check another settings if everything is ok. Then I will upgrade on "PROD" server. :)

Thank you!
